mkosi/resources/mkosi-obs/mkosi.conf

```diff
@@ -2,8 +2,18 @@
 [Build]
 SandboxTrees=/usr/src/packages/SOURCES:/usr/src/packages/SOURCES
 
+[Build]
+ToolsTree=
+History=no
+CacheDirectory=
+Incremental=no
+
 [Output]
+OutputDirectory=
```

Comment:
We should set `Checksum=yes` here as well, I think we want to move towards a world where all the OBS specific configuration is in here and not in OBS itself?

Reply:
And the other ones from obs-build as well:

```
        --repository-key-check=no \
        --with-network=never \
        --local-mirror file:///.build.binaries/ \
        --cache /.build.binaries/ \
```